Lib/test/seq_tests.py

```diff
@@ -321,6 +321,7 @@ def test_repeat(self):
             self.assertEqual(self.type2test(s)*(-4), self.type2test([]))
             self.assertEqual(id(s), id(s*1))
 
+    @unittest.skip("TODO: RUSTPYTHON")
```

Comment:
if this 32bit specific issue, adding 32bit note with RUSTPYTHON would be helpful